runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -282,4 +285,14 @@ private void drawTooltipIfMouseover(Graphics2D graphics, XpGlobe mouseOverSkill,
 		xpTooltip.render(graphics);
 		graphics.translate(-x, -y);
 	}
+
+	private String getActionKillsText(String skillName)
+	{
+		List<Skill> COMBAT = Arrays.asList(Skill.ATTACK, Skill.STRENGTH, Skill.DEFENCE, Skill.RANGED, Skill.HITPOINTS);
```

Comment:
Don't we have a combat skills list somewhere already?
Like before this PR was made